XenModel/Messages.resx

```diff
@@ -10698,7 +10692,7 @@ Do you want to connect to the pool master '{1}'?</value>
     <value>Deleting snapshots ...</value>
   </data>
   <data name="SNAPSHOT_FREQUENCY" xml:space="preserve">
-    <value>Snapshot schedule</value>
+    <value>Snapshot Schedule</value>

```

Comment:
This change (and the one on VMSS_VMS) requires a fix in the NewPolicyWizardTest.